newIDE/app/src/ObjectsList/index.js

```diff
@@ -827,9 +839,6 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
                     onEditItem={objectWithContext =>
                       onEditObject(objectWithContext.object)
                     }
-                    onAddNewItem={onAddNewObject}
```

Comment:
This had a "smart" behavior: when the width was too small, the button would collapse by not showing its label anymore, but just the + icon. Something important to keep for small screens I think?

Reply:
Yeah I noticed that, we can keep that indeed